src/transformers/dynamic_module_utils.py

```diff
@@ -383,7 +385,7 @@ def get_cached_module_file(
             new_files.append(module_file)
 
     except OSError:
-        logger.error(f"Could not locate the {module_file} inside {pretrained_model_name_or_path}.")
+        logger.info(f"Could not locate the {module_file} inside {pretrained_model_name_or_path}.")
```

Comment:
We were throwing an exception and a warning 🤔

Lowered the severity, otherwise we were throwing warnings in most model `from_pretrained` (see changes in `modeling_utils.py`)